data/registry/instrumentation-go-gocql.yml

```diff
@@ -14,3 +14,4 @@ authors:
 urls:
   repo: https://go.opentelemetry.io/contrib/instrumentation/github.com/gocql/gocql
 createdAt: 2022-12-07
+isFirstParty: true
```

Comment:
that repo URL is incorrect, but also I don't think this is first party

Reply:
What would be the correct URL? And how do you find it?